interfaces.d.ts

```diff
@@ -26,11 +26,28 @@ export interface PolymerInit {
   extends?: string;
   properties?: PolymerElementProperties;
   observers?: string[];
-  template?: HTMLTemplateElement|string;
+  _template?: HTMLTemplateElement;
   hostAttributes?: {[key: string]: any};
   listeners?: {[key: string]: string};
+  behaviors?: BehaviorInit | BehaviorInit[];
+
+  // Lifecycle methods
+  registered?(): void;
+  created?(): void;
+  attached?(): void;
+  detached?(): void;
+  ready?(): void;
+  attributeChanged?(name: string, old?: string, value?: string);
```

Comment:
This lacks a return type. Please see the CI output for the full error. You can run `npm run generate-types` to locally run the same check.

Reply:
Thanks!